.travis.yml

```diff
@@ -76,6 +76,10 @@ matrix:
     - dist: trusty
       env:
         - JOB="3.6_DOC" DOC=true
+    # In allow_failures
+    - dist: trusty
+      env:
+        - JOB="3.6_ASV" ASV=true
```

Comment:
Could you put this just before the docs build? (I think I will more refer to the doc build, and then it is easier to say the last build :-))